### ISSUE TITLE
New Steam UI

### ISSUE THREAD

**Issue by Vovkiv** [tkashkin/Adwaita-for-Steam#151]
Today was update with new UI for Steam, so skins doesn't work.
Also, it seems, there no more option for changing skins?
Did i miss something or Steam doesn't support skins anymore?

**Comment by TerohsLab** [comment]
Love when people just close things with "works for me" without explaining how :(

**Comment by Vovkiv** [comment]
@TerohsLab since you probably on Linux, you could install program https://github.com/Foldex/AdwSteamGtk (which is manager for this specific skin) and install theme with it.
If it doesn't work for you, open issues on program’s github page.

**Comment by Vovkiv** [comment]
I think it related to https://github.com/tkashkin/Adwaita-for-Steam/issues/144

**Comment by TerohsLab** [comment]
Yeah, i figured it would break after that big steam update. I'm just generally annoyed  by people closing issues with "works for me" without any explanation.

And yeah i was searching like crazy for the "skins" option in interface and was thinking i was going insane.

I'll just wait .. the new design isn't that bad.

And yes im on Manjaro KDE.

**Comment by Vovkiv** [comment]
> Yeah, i figured it would break after that big steam update. I'm just generally annoyed by people closing issues with "works for me" without any explanation.

@TerohsLab 
 Well, i mean, i was feeling like this was stupid on my side, since what i have done is simply reinstall theme, because, you know, steam got big update, files was replaced with new one, so it should be obvious to reinstall skin, before claiming that it didn't work.
So i created issue before i reinstalled theme and was like "Oh..."

**Comment by TerohsLab** [comment]
Hehehe .. dont worry happens to all of us.

Just recently i was wildly complaining to co-workers how a very important program from my PC went "missing" overnight.

Admin checked the log and it was me who manually removed it the day before.

I went full "Uhhhhhhhh -.-#" *facepalm.picard*

**Comment by Vovkiv** [comment]
> Just recently i was wildly complaining to co-workers how a very important program from my PC went "missing" overnight.

Let me guess, it was Friday night?
Because, for some reasons, stuff like this happens always on Friday evening/night
